daq_macros.py

```diff
@@ -3565,7 +3565,12 @@ def topViewCheckOn():
   setBlConfig(TOP_VIEW_CHECK,1)
 
 def anneal(annealTime=1.0):
-  if daq_utils.beamline == 'fmx':
+  if daq_utils.beamline == 'fmx' or daq_utils.beamline == 'amx':
```

Comment:
can you also add a TODO that all of this governor handling code can be removed to use standard code in LSDC2?

Reply:
Yup, done.